capstone/src/main/webapp/profile-loader.js

```diff
@@ -76,11 +69,12 @@ function createClubElement(text) {
 function saveProfileChanges() {
   const newYear = document.getElementById("edit-year").innerHTML;
   const newMajor = document.getElementById("edit-major").innerHTML;
+  const newName = document.getElementById("edit-name").innerHTML;
 
-  var updateProfile = document.getElementById("update-profile");
-
-  updateProfile.innerHTML = '<input type="hidden" name="new-year" value="' + newYear + '">';
-  updateProfile.innerHTML += '<input type="hidden" name="new-major" value="' + newMajor + '">';
+  //var updateProfile = document.getElementById("update-profile");
```

Comment:
Should this be deleted? Or not commented out?

Reply:
Ah, yes just deleted it. Thanks for catching this!